lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -334,18 +328,22 @@ public Object run() throws Exception {
           });
 
       LOG.log(Level.FINE, "YARN registration: register AM at \"{0}:{1}\" tracking URL \"{2}\"",
-          new Object[] {AM_REGISTRATION_HOST, AM_REGISTRATION_PORT, this.trackingUrl});
+          new Object[] {this.trackingURLProvider.getHost(),
```

Comment:
Can implement `HttpTrackingURLProvider.toString()` to print all three parameters at once